### ISSUE TITLE
Validation failed if the badges contains data url instead of url in image field of badge

### ISSUE THREAD

**Issue by danishfastian** [IMSGlobal/openbadges-validator-core#203]
If we are using an assertion where image field contains the link of png file than assertion get validated successfully. However if we replace image with the image url we are getting error.

Following are sample assertion the one with error is generating error.

[samplejson.zip](https://github.com/IMSGlobal/openbadges-validator-core/files/2000553/samplejson.zip)

**Comment by ottonomy** [comment]
Thanks! Fixed in 46e73fa0d08706f52bb3aa5fb03fd21cf9c50b04 to be pushed soon. Relates to #186.

**Comment by danishfastian** [comment]
Thanks for fixing the issue.  I am still facing the issue, any idea when it will be pushed to live validator? 

Just to clarify, when I get the code and clean install it locally, open badge validator console application validates successfully.
However hosted validators at https://openbadgesvalidator.imsglobal.org/ and https://badgecheck.io/ are still unable to validate.

**Comment by mgylling** [comment]
Hi @danishfastian. The update to 1.1 was released on PyPi today. The IMS vaildator will be updated later today, and badgecheck most probably very soon as well. 

https://pypi.org/project/openbadges/#history

**Comment by ghost** [comment]
Seems to validate on imsglobal but the fix doesn't appear to be updated on badgecheck. 

Getting the error
```
<class 'requests.exceptions.InvalidSchema'> ["InvalidSchema: No connection adapters were found for 'data:image/png;base64,iVBORw0KGgoAA...
```

**Comment by jbohrer** [comment]
Resolved for IMS Validator in #211
